editors/micro/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/zyedidia/micro 2.0.4 v
+
+homepage            https://micro-editor.github.io/
+
+description         A modern and intuitive terminal-based text editor
+
+long_description    micro is a terminal-based text editor that aims to be \
+                    easy to use and intuitive, while also taking advantage of \
+                    the capabilities of modern terminals. As its name \
+                    indicates, micro aims to be somewhat of a successor to \
+                    the nano editor by being easy to install and use. Micro \
+                    supports a full-blown plugin system. Plugins are written \
+                    in Lua and there is a plugin manager to automatically \
+                    download and install your plugins for you.
+
+categories          editors
+license             MIT
+
+build.target        github.com/go-delve/delve/cmd/dlv
+
+maintainers         {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
+
+build.cmd           make
```

Comment:
Maybe use the makefile portgroup ?

Reply:
We don't need all the things that the `make` portgroup provides.  Most of what we need for this port is provided by the `golang` portgroup.  We call make ourselves because the project comes with its own `Makefile`, as opposed to the `golang` portgroup's default of invoking `go build` directly.